pygeoapi/provider/rasterio_.py

```diff
@@ -174,7 +174,8 @@ def query(self, properties=[], subsets={}, bbox=None, bbox_crs=4326,
         :returns: coverage data as dict of CoverageJSON or native format
         """
 
-        bands = properties
+        bands = properties or []
+        subsets = dict(subsets) if subsets is not None else {}
```

Comment:
I guess not, it is even a bit inconsistent, since I did not make a copy for the `properties` parameter, but I'm making a copy for `subsets` 😅. I'll modify it